DB/NEW_KT_DB/DataAccess/ObjectManager.py

```diff
@@ -12,34 +12,56 @@ def __init__(self, db_file: str):
     # for internal use only:
 
     # Riki7649255 based on rachel-8511
-    def create_management_table(self, table_name, table_structure='object_id INTEGER PRIMARY KEY AUTOINCREMENT,type_object TEXT NOT NULL,metadata TEXT NOT NULL')
+    def create_management_table(self, table_name, table_structure='object_id INTEGER PRIMARY KEY AUTOINCREMENT,type_object TEXT NOT NULL,metadata TEXT NOT NULL'):
+        self.db_manager.create_table(table_name, table_structure)
+    
+    def create_management_table_with_str_id(self, table_name, table_structure='object_id TEXT NOT NULL PRIMARY KEY ,metadata TEXT NOT NULL'):
         self.db_manager.create_table(table_name, table_structure)
 
     
     # Riki7649255 based on saraNoigershel
-    def insert_object_to_management_table(self, table_name, object):
-        self.db_manager.insert_data_into_table(table_name, object)
+    def insert_object_to_management_table(self, table_name, object_metadata):
+        self.db_manager.insert_data_into_table(table_name, ['metadata'], (object_metadata))
+
+    def insert_object_to_management_table_with_str_id(self, table_name, id, metadata):
+        self.db_manager.insert_data_into_table(table_name, ['object_id','metadata'],(id, metadata))
 
 
     # Riki7649255 based on rachel-8511
     def update_object_in_management_table_by_criteria(self, table_name, updates, criteria):
         self.db_manager.update_records_in_table(table_name, updates, criteria)
+    
```

Comment:
What if pk column name is different?

Reply:
@riki7649255 should pk_col_name be sent as a parameter, or do all tables have to have object_id is the pk column name by convention?